src/js/infragistics.loader.js

```diff
@@ -609,7 +609,26 @@ $.ig.dependencies = [
 					{ name: "Interactivity", parentWidget: "igDataChart" },
 					{ name: "VisualData", parentWidget: "igDataChart" }]
 	},
-
+	{
+		widget: "igFinancialChart",
+		dependency: [
+			{ name: "_ig_dv_dataseriesadapter" },
+			{ name: "_ig_dv_geometry" },
+			{ name: "_ig_dv_extendedaxes" },
+			{ name: "_ig_dv_commonwidget"},
+			{ name: "_ig_simple_datachart_core"},
+			{ name: "_ig_datachart_domainchart"},
+			{ name: "_ig_datachart_categorycore" },
+			{ name: "_ig_datachart_interactivity" }
+		],
+		scripts: [
+			"$path$/modules/infragistics.financialchart.js",
+			"$path$/modules/infragistics.ui.financialchart.js"
+		],
+		css: [ "$path$/structure/modules/infragistics.ui.financialchart.css" ],
```

Comment:
@davidnegley Is this CSS only for the sample or it's general styling for the igFinancialChart?

Reply:
it's general styling for the igFinancialChart.